week02/code/PriorityQueue.cs

```diff
@@ -1,28 +1,31 @@
-﻿public class PriorityQueue {
+﻿public class PriorityQueue
+{
     private List<PriorityItem> _queue = new();
 
     /// <summary>
     /// Add a new value to the queue with an associated priority.  The
-    /// node is always added to the back of the queue irregardless of 
+    /// node is always added to the back of the queue regardless of 
     /// the priority.
     /// </summary>
     /// <param name="value">The value</param>
     /// <param name="priority">The priority</param>
-    public void Enqueue(string value, int priority) {
+    public void Enqueue(string value, int priority)
+    {
         var newNode = new PriorityItem(value, priority);
         _queue.Add(newNode);
     }
 
-    public String Dequeue() {
+    public string Dequeue()
+    {
         if (_queue.Count == 0) // Verify the queue is not empty
         {
-            Console.WriteLine("The queue is empty.");
-            return null;
+            throw new InvalidOperationException("The queue is empty.");
```

Comment:
Where we used to log errors, we now throw exceptions so we can unit tests that exceptions were thrown.